packages/composite-checkout/README.md

```diff
@@ -0,0 +1,559 @@
+# WP Checkout
```

Comment:
This title could be same as package name - WP prefix feels confusing here.

Reply:
Oops. That's left over from the original package name. Thanks!